integration-tests/testconfig/README.md

```diff
@@ -57,6 +57,8 @@ This file is recommended for local use to adjust dynamic variables or modify pre
 version = "your tag"
```

Comment:
@smartcontractkit/test-tooling-team have we validated this approach still works? the overriden version key here was not applied when I ran smoke testing locally.

Reply:
@patrickhuie19 yes, this should work. I just checked it by running integration-tests/smoke/ocr_test.go with custom version in overrides.toml. See screenshot below:

<img width="1984" alt="image" src="https://github.com/user-attachments/assets/eb626d9b-ff09-47ce-8240-6a5f2fe9e1ba">

Reply:
@patrickhuie19 yes, this should work. I just checked it by running integration-tests/smoke/ocr_test.go with custom version in overrides.toml. See screenshot below:

<img width="1819" alt="image" src="https://github.com/user-attachments/assets/2dc5310b-b94b-40bb-b843-193e037f43c5">